protocol/clproto_cpp/test/tests/include/test_clproto/encode_decode_helper.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
```

Comment:
nitpick: instead of `include/test_clproto/encode_decode_helper.hpp`, I would prefer this file to simply be `include/test_encode_decode.hpp`. Because this header is only included for test builds, I think it's nicer to just have it at the top level and named after the function it contains. Besides, `clproto.h` is also included at the top-level.

```c++
#include "test_clproto/encode_decode_helper.hpp"
```